### PR TITLE
Added that vCenter Server HA is not supported

### LINKED ISSUE #1811: Document that vCenter Server HA is not supported
Per internal bug 2130436 we don't document whether or not we support vCenter Server HA, which is a new feature introduced in 6.5, and is not to be confused with vSphere HA (which we do support).

Engineering investigations are in internal bug 2127492.

### PR DESCRIPTION
**Pull request by vidya-v**
Documented that vCenter Server HA is not supported.

Hi @stuclem , can you please review this PR? Thanks!

Fixes #1811

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/interop.md [hunk 1]
```diff
@@ -58,4 +58,5 @@ vSphere Integrated Containers Engine does not currently support the following vS
...
 - vSphere Fault Tolerance: vSphere Integrated Containers does not implement vSphere Fault Tolerance. However, VCH processes that stop unexpectedly do restart automatically, independently of vSphere Fault Tolerance.
 - vSphere Virtual Volumes&trade;: You cannot use Virtual Volumes as the target datastores for image stores or volume stores.
-- Snapshots: Creating and reverting to snapshots of the VCH endpoint VM or container VMs can cause vSphere Integrated Containers Engine not to function correctly.
\ No newline at end of file
+- Snapshots: Creating and reverting to snapshots of the VCH endpoint VM or container VMs can cause vSphere Integrated Containers Engine not to function correctly.
+- vCenter Server High Availabilty: You cannot deploy VCHs on a highly available vCenter Server.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/interop.md]: Actually, I just noticed and saw that the official feature name is vCenter High Availability, not vCenter Server HA: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.avail.doc/GUID-4A626993-A829-495C-9659-F64BA8B560BD.html

My mistake!

**Comment by vidya-v** [comment]
Thanks, @stuclem ! Made the change.

**Comment by stuclem** [comment]
Also, there is some confusion about the distinction between vCenter HA and vSphere HA, which I only discovered recently are two different things. Maybe we should add a note to the vCenter HA section saying that vSphere HA _is_ supported, linking to the vSphere HA section earlier in the same topic.

**Comment by vidya-v** [comment]
@stuclem , I added a link about vSphere HA support. Is the change okay? Can I merge it?

**Pull request merged**